### PR TITLE
WIP: Modified language to adhere to style guide

### PR DESCRIPTION
**Pull request by jc-berger**
Fixes [#254](https://github.com/devfile/api/issues/254) Review Language in Authoring Stacks.

Since I am doing a review of all the authoring stacks docs, I'll be making several PRs. I don't want any single PR to be too large, so I'll make a new PR for every several docs that I modify to meet the IBM Style Guide's requirements.

See related PRs, #57 and #59

### CODE REVIEW

**Comment by jc-berger** [comment]
See corresponding Jira issue: https://issues.redhat.com/browse/RHDEVDOCS-2565

File: docs/modules/user-guide/partials/proc_adding-container-component-to-a-devfile.adoc [hunk 1]
```diff
@@ -52,9 +52,9 @@ components:
       command: ['sleep', 'infinity']
 ----
 +
-It specifies the type of the component, `container` and the `image` attribute names the image to be used for the component using the usual Docker naming conventions, that is, the above `image` attribute is equal to `docker.io/library/golang:latest`.
+Specify the type of component with the `container`. Use the `image` attribute to name the image used for the component. When naming the `image` use Docker naming conventions; for example, the `image` attribute is equal to `docker.io/library/golang:latest`.
```
Comment (boczkowska) [inline docs/modules/user-guide/partials/proc_adding-container-component-to-a-devfile.adoc:55]:
```suggestion
Specify the type of component with the `container` attribute. Use the `image` attribute to name the image used for the component. When naming the `image` attribute use Docker naming conventions; for example, the `image` attribute in the example above is equal to `docker.io/library/golang:latest`.
```

File: docs/modules/user-guide/partials/proc_adding-container-component-to-a-devfile.adoc [hunk 2]
```diff
@@ -52,9 +52,9 @@ components:
...
 +
-A `container` component has many features that enable augmenting the image with additional resources and information needed for meaningful integration of the *tool* provided by the image with {prod}.
+A `container` component has many features so you can augment the image with additional resources and information needed for integration of the tools provided by the devfile image.
```
Comment (boczkowska) [inline docs/modules/user-guide/partials/proc_adding-container-component-to-a-devfile.adoc:57]: I feel like this wording is slightly confusing. Maybe smth along the lines of "With a `container component, you can augment..."

File: docs/modules/user-guide/partials/proc_referring-to-a-parent-devfile-in-a-devfile.adoc [hunk 3]
```diff
@@ -36,7 +36,7 @@ parent:
 +
 .Parent referred by uri
 ====
-Using the URI if it has been published on a static http server (like gist or pastebin).
+Using the URI if it has been published on a static http server, like `gist` or `pastebin`.
```
Comment (boczkowska) [inline docs/modules/user-guide/partials/proc_referring-to-a-parent-devfile-in-a-devfile.adoc:39]:
```suggestion
Using the URI if it has been published on a static http server, such as `gist` or `pastebin`.
```
(IBM styleguide recommendation)

File: docs/modules/user-guide/partials/proc_referring-to-a-parent-devfile-in-a-devfile.adoc [hunk 4]
```diff
@@ -67,7 +67,7 @@ parent:
 [role="_additional-resources"]
 .Additional resources
 
-For more information on referring to parent devfiles, go to the following topics:
+For more information on referring to parent devfiles, see:
```
Comment (boczkowska) [inline docs/modules/user-guide/partials/proc_referring-to-a-parent-devfile-in-a-devfile.adoc:70]:
```suggestion
For more information about referring to parent devfiles, see:
```

(Again, IBM rule. )

**Comment by jc-berger** [comment]
Okay, so I've made a new commit, addressing Yana's feedback and including more docs that I edited. But I don't know why the doc, docs/modules/user-guide/attachments/jsonschemas/next/devfile.json, shows as a doc that's been edited. I haven't touched this doc...

I did have an issue pushing my new commit, so I fixed the rebase problem. I'm thinking that's how this doc got thrown into the mix.

**Comment by jc-berger** [comment]
Since I had rebase issues with this PR, I've made a new PR, #67 

All the content changes reflected in this PR has been moved over to #67. I'm closing this issue since we'll work on and merge #67.

**Pull request closed without merging**